crates/astria-composer/src/searcher.rs

```diff
@@ -197,6 +189,12 @@ impl Searcher {
 
     /// Serializes and signs a sequencer tx from a rollup tx.
     fn handle_pending_tx(&mut self, rollup_tx: Transaction) {
+        use astria_sequencer::{
+            accounts::types::Nonce,
+            sequence,
+            transaction::action,
+        };
```

Comment:
are you planning to eventually break these types out as well?

Reply:
Yes, that will be done in a follow-up so that this PR doesn't explode.